### PR TITLE
Show notification popup after application start in tray

### PR DESCRIPTION
**Pull request by thepurple**

### CODE REVIEW

File: src/core/controller.cpp [hunk 1]
```diff
@@ -226,6 +226,12 @@ void Controller::enableTrayIcon() {
     };
     connect(m_trayIcon, &QSystemTrayIcon::activated, this, trayIconActivated);
     m_trayIcon->show();
+    if (!ConfigHandler().startupLaunchValue()) {
```
Comment (StyleT) [inline src/core/controller.cpp:229]: This logic won't work if I have "auto start" feature enabled & launch app manually.

File: src/core/controller.cpp [hunk 2]
```diff
@@ -226,6 +226,12 @@ void Controller::enableTrayIcon() {
...
+        m_trayIcon->showMessage("Flameshot",
+                                QObject::tr("Application is started successfully"),
```
Comment (StyleT) [inline src/core/controller.cpp:231]:
```suggestion
                                QObject::tr("Hello, I'm here! Click icon in the tray to take a screenshot or click with a right button to see more options."),
```

Comment (thepurple) [reply src/core/controller.cpp]: I did my own commit with translations to keep git history more clear

File: src/core/controller.cpp (hunk 1 above)
Comment (thepurple) [reply src/core/controller.cpp]: Logic is completely changed

File: src/config/geneneralconf.cpp [hunk 3]
```diff
@@ -213,6 +217,19 @@ void GeneneralConf::initAutostart() {
             &GeneneralConf::autostartChanged);
 }
 
+void GeneneralConf::initShowStartupLaunchMessage() {
+    m_showStartupLaunchMessage = new QCheckBox(tr("Show startup message on launch"), this);
```
[the remark this answers is not in the document]
Comment (thepurple) [reply src/config/geneneralconf.cpp]: Done

**Pull request merged**